mozilla/blurts-server
# MNTOR-2248: 1 click deploy to prod

# References:

Jira: MNTOR-2248



# Description
One click deploy to production

# How to test
* Select environment when running the workflow
* Put in the sha or release tag that was deployed prior. Please make sure to double check the spelling matches exactly

# Checklist (Definition of Done)
